common/BasicTypes.acn

```diff
@@ -23,11 +23,141 @@
 
 BasicTypes DEFINITIONS ::= BEGIN
 
-PUSC-BOOL [false-value '0'B]
+PUSC-BOOL1 [false-value '0'B] --PFC-0
+PUSC-BOOL8 [false-value '00000000'B] --PFC-8
+PUSC-BOOL16 [false-value '0000000000000000'B] --PFC-16
+PUSC-BOOL32 [false-value '00000000000000000000000000000000'B] --PFC-32
 
-PUSC-UINT [size 64, encoding pos-int]
+PUSC-UINT4 [size 4, encoding pos-int] --PFC-0
```

Comment:
does ptc/pfc define endianess?

Reply:
It is mentioned only in UINT chapter that bit 0 is MSB

Reply:
so maybe set that endianness everywhere, just in case